src/main/java/net/mcreator/ui/dialogs/tools/WoodPackMakerTool.java

```diff
@@ -160,8 +161,8 @@ private static void addWoodPackToWorkspace(MCreator mcreator, Workspace workspac
 		woodBlock.resistance = 2.0 * Math.pow(factor, 0.8);
 		woodBlock.destroyTool = "axe";
 		woodBlock.breakHarvestLevel = 0;
-		woodBlock.plantsGrowOn = true;
 		woodBlock.flammability = (int) Math.round(5 * factor);
+		woodBlock.rotationMode = 5; // log rotation

```

Comment:
Does wood need log rotation?

Reply:
As bark blocks (wood) have the same texture everywhere, rotation is not needed.

Reply:
Vanilla wood block do use [log rotation ](https://minecraft.fandom.com/wiki/Wood#Block_states)

Reply:
Hmm, didn't know that :D